packages/node/src/types.ts

```diff
@@ -0,0 +1,49 @@
+import { Address, Node } from "@counterfactual/types";
+
+// TODO: the contents of this file should probably be in the types package
```

Comment:
@snario thoughts here?

Reply:
If it is not used in any other package, I'd just keep it in this package. I think the requirement for `@counterfactual/types` should be "it is used in >= 2 other packages"

Reply:
But it'll be used by the playground, as it'll need to know what events to register callbacks for and what arguments those callbacks receive.